### DISCUSSION TITLE
Share slides without download option

### DISCUSSION THREAD

**Discussion by bgeisemeyer** [excalidraw/excalidraw#8292] [Q&A]
Hey there, 

i wonder if it is possible to share slides via read-only link or embedding without making it possible to download the slides?!
I sometimes would like to share slides to read, but don't want them to be storable and shareable. The content adjusts over time, so stored and shared copies would not be up to date, whereas the online link would be. This can be crucial.

Is there an option or might there be a an issue for that? 
On my research I have not found one.

Thanks :)

**Comment by maielo** [comment]
Sounds reasonable, we are planning to improve presentations in general :) 

Going to add it to our roadmap.

Thanks!

**Reply by bgeisemeyer** [reply]
You are welcome. :)
